README.md

```diff
@@ -42,6 +42,11 @@ Hinweise zur Synchronisation im Frontend
 * Damit die Synchronisation nach dem Speichern direkt im Frontend funktioniert, muss die Seite im Frontend über die selbe Domain aufgerufen werden, mit welcher man sich im Backend eingeloggt hat, da ansonsten die Backend-Session nicht mit dem Frontend übereinstimmt (Beispiel: im Backend mit www. eingeloggt aber das Frontend ohne www. aufgerufen). Selbes gilt in Multidomain-Umgebungen und für http/https. 
 
 
+Fehlerbehebung
+---------
+Falls die Synchronisation von aktualisierten Dateien fehlschlägt, kann der Grund ein falscher Timestamp sein. Das `updatedate` in der Datenbank muss älter sein als der Zeitstempel der hochgeladenen Datei.
```

Comment:
was wäre die lösung für das problem? dies sollte hier auch benannt sein

Reply:
ich schreibe den lösungsweg gerne mit rein, allerdings denke ich das das zu speziell für die readme ist.

in meinem fall sind bei einem serverwechsel die zeitstempel der .yml dateien in den ordnern 
'module' und 'templates' verändert worden, so das zeitstempel, die sehr weit in der zunkunft liegen in der db gelandet sind...

@alexplusde meinte, das in der vergangenheit bereits häufiger probleme mit dem zeitstempel gab. ob die allerdings die gleiche ursache hatten weiß ich natürlich nicht?

was meint ihr?

Reply:
Im Grunde will man vermutlich immer den Zeitstempel in der Datenbank in die Vergangenheit setzen. 

Es kommt aber darauf an, was genau gerade falsch läuft. Wenn Dateien mit altem Zeitstempel hochgeladen werden, muss man evtl. sein FTP-Programm anders konfigurieren. Wenn der Server die falsche Zeitzone eingestellt hat und das Problem dauerhaft ist, muss der Hoster eine Korrektur am Server vornehmen.

Reply:
@isospin kannst du hier nochmal einen Textentwurf vorschlagen?